cli/cli.go

```diff
@@ -626,6 +626,25 @@ func Priam(args []string, defaultCfgFile string, infoW, errorW io.Writer) {
 			Description: "Supported types are User, Group, Role, PasswordState, ServiceProviderConfig\n",
 			Action:      cmdWithAuth1Arg(cfg, CmdSchema),
 		},
+		{
+			Name: "token", Usage: "token operations",
```

Comment:
I did not know that. Since I saw "schema" at the end, after "user". Should I move that one too?